File: Bank/src/main/Retiro.java
package main;

import java.util.InputMismatchException;
import java.util.Scanner;

public class Retiro {
    public static float retiro (float saldo){
        float nSaldo=0;
        float cantidad;
        System.out.println("Su saldo es de: $"+(int)saldo+"\nQue cantidad desea retirar(Centavos no permitidos)?");
        Scanner scan = new Scanner(System.in);
        try {
            cantidad = scan.nextFloat();
            if(cantidad < 0){
                System.out.println("Operacion imposible\nRegresando a Menu");
            }else if(cantidad > saldo){
                System.out.println("Fondos insuficientes");
            }
            else{
                nSaldo = saldo - cantidad;
            }
        }catch (InputMismatchException e){
            System.out.println("No se permiten centavos en sus operaciones");
        }
        return nSaldo;
    }
}

File: BankPy/Retiro.py
def retiro(saldo):
    nsaldo = 0
    print(f"Su saldo es de: ${int(saldo)}\nQue cantidad desea retirar(Centavos no permitidos, seran redondeados)?")
    cantidad = float(input("Cantidad:"))
    if cantidad < 0:
        print("Operacion imposible\nRegresando a Menu")
    elif cantidad > saldo:
        print("Fondos insuficientes")
    else:
        nsaldo = saldo - cantidad
    return nsaldo
